config/routes.rb

```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
-  resource :budgets, only: [:show, :new, :create, :edit, :update, :destroy]
+  resources :budget do
+    resources :incomes
+    resources :expenses
+    resources :goals
```

Comment:
Do `goals` exist?
Also, would any of the paths in the app/spec will need to be updated with this change?

Reply:
No goals yet, so I will remove...will have a look at the paths, thanks Elle.